### PR TITLE
Improve Runestone testing

### PR DESCRIPTION
**Pull request by bjones1**
This improves timed exam unit tests, as a first step toward #1146. Next up will be `selectquestion` testing.

Add: more testing for a timed exam.
Fix: Improve unit testing approach:
       Run webpack before each test
       Properly close the Selenium driver
       Provide a wait_until_ready method

### CODE REVIEW

**Comment by bjones1** [comment]
@bnmnetp, it looks like the unit tests are confused here too -- Py 3.7 passes, Python 3.8 doesn't. We're definitely in need of some unit test improvements.

**Comment by bjones1** [comment]
FYI, this passed on Travis (https://travis-ci.org/github/bjones1/RunestoneComponents/builds/763045835).

**Comment by bjones1** [comment]
Yikes. Apparently, a delay makes the tests pass...

...which is why we need a better test methodology. The new code allows components to indicate when they're ready. I didn't include it in this fix, since a shortanswer question is used by selectquestion and is a big, potentially breaking change. When this is merged, I'll submit a PR to update the poll component to use this "smart" approach. I chose pool because the poll component is a low-risk place to start.

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +19,6 @@ script:
 - "python -m pip install -U pip"
 - "pip install -e ."
 - "pip install -U -r requirements-dev.txt"
-- "npm run build"
```
Comment (bnmnetp) [inline .travis.yml:22]: This file could be deleted since I moved us over to using github actions.

**Pull request merged**